packages/ui-date-time-input/src/DateTimeInput/index.tsx

```diff
@@ -64,8 +64,7 @@ class DateTimeInput extends Component<DateTimeInputProps, DateTimeInputState> {
     dateFormat: 'LL' // Localized date with full month, e.g. "August 6, 2014"
   } as const
 
-  context!: React.ContextType<typeof ApplyLocaleContext>
-
+  declare context: React.ContextType<typeof ApplyLocaleContext>
```

Comment:
This is the new recommended syntax for `context` (which is required for React 18)